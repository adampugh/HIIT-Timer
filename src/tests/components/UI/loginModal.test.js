import React from "react";
import { shallow } from "enzyme"; 

import LoginModal from "../../../components/UI/loginModal";

test("should render loginModal correctly", () => {
    const wrapper = shallow(<LoginModal />);
    expect(wrapper).toMatchSnapshot();
});