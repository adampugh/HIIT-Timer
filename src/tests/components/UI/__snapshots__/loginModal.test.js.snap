// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loginModal correctly 1`] = `
<Modal
  appElement={null}
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={200}
  contentLabel="login modal"
  isOpen={false}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
  style={
    Object {
      "content": Object {},
      "overlay": Object {
        "background": "rgba(0, 0, 0, 0.6)",
        "zIndex": 2000,
      },
    }
  }
>
  <div
    className="modal__box"
  >
    <div
      className="modal__header"
    >
      <div />
      <div>
        <h3>
          Log In
        </h3>
      </div>
      <div
        className="modal__close"
      >
        <h3>
          <span>
            <i
              className="fas fa-times"
            />
          </span>
        </h3>
      </div>
    </div>
    <hr />
    <button
      className="modal__login-button"
    >
      <img
        alt="google logo"
        src="googlelogo.jpg"
      />
      <span>
        Sign in with Google
      </span>
    </button>
    <button
      className="modal__login-button"
    >
      <img
        alt="facebook logo"
        src="facebooklogo.jpg"
      />
      <span>
        Sign in with Facebook
      </span>
    </button>
  </div>
</Modal>
`;
